components/perl/Mail-SPF/perl-integrate-module.conf

```diff
@@ -0,0 +1,34 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Friedrich Kink
+#
+
+%hook-no-license%
+printf "%s\n" "$COPYRIGHT" > "$DISTRIBUTION.license"
+LICENSE="MPL-1.1"
```

Comment:
The license is definitely not MPL-1.1.  It looks like it is some kind of BSD, but you need to find which one: https://spdx.org/licenses/

Reply:
> No I don't, the author expressed his thoughts in a file called LICENSE which we should respect. Assuming something else without feedback from the author is simply wrong. I'll remove of course MPL-1.1 but the Mail-SPF.license file contains only the original license file.

Sorry, I do not understand.  Please elaborate.  Do you agree that the `LICENSE` file (and `Mail-SPF.license` file) contains some kind of BSD-style license?

Reply:
This section is part of almost any license terms. And "some kind of" is exactly what I mean with assumption.

Reply:
> This section is part of almost any license terms. And "some kind of" is exactly what I mean with assumption.

What you refer to with "this section"?

Sorry, please elaborate about "assumption" more.  I'm afraid I still do not understand what you try to say by all of that.

And, I still miss answer to my question: Do you agree that the `LICENSE` file (and `Mail-SPF.license` file) contains some kind of BSD-style license?

If I know the answer to that question maybe I'd better understand what problem you see here and there.

Thanks.

Reply:
With "this section" I mean the stanza in capital letters in the license file. 
License is a strong legal term. If you change the license because you think it contains "some kind of" what ever other license without asking for feedback from the original author(s) you are liable for the consequences. Just a simple example:
If the license just contains "You can use the source for what ever you want but you are by any means not allowed to change the code" and you replace it because it contains "some kind of" whatever known license and this 15 page license contains a small stanza stating "in case of a bug you are allowed to fix it", you turn it into the opposite of what the author(s) intended.  And now, who is liable in case of a legal proceedings?

Basically it is not about whether I agree to your statement "contains some kind of BSD-style license". It is about we are not allowed to change whatever license.